src/Couchbase.Lite.Shared/Replication/RemoteSession.cs

```diff
@@ -477,16 +513,24 @@ internal void SendAsyncMultipartRequest(HttpMethod method, string relativePath,
             var message = new HttpRequestMessage (method, url);
             message.Headers.Add ("Accept", "*/*");
 
+#if __IOS__  // hack to get a client with NSUrlSessionHandler which is much faster
+            var client = _iosHttpClient;
```

Comment:
Would it do this automatically if we switched our project's iOS build settings for HttpClient from Managed(default) to NSUrlSession? Then we wouldn't need this part.